tests/dummy/config/targets.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-env node */
+
+module.exports = {
+  browsers: [
```

Comment:
yes, 2.13 uses this along with babel-preset-env to only transpile the things that are required to be transpiled for the given browser matrix